src/osd/PrimaryLogPG.cc

```diff
@@ -8328,6 +8328,7 @@ int PrimaryLogPG::prepare_transaction(OpContext *ctx)
   // prepare the actual mutation
   int result = do_osd_ops(ctx, *ctx->ops);
   if (result < 0) {
+    dout(10) << " do_osd_ops error: " << cpp_strerror(result) << dendl;
```

Comment:
i'd suggest move this into `PrimaryLogPG::do_osd_ops()`.

Reply:
i'v recommitted follow your advice, please review.